### PR TITLE
Support paginator for parameters and empty descriptions in moved to root lambda 

### PR DESCRIPTION
**Pull request by rickardl**
The below change hopefully resolves all issues we encountered in #272 

* support larger number of entries of SSM parameters by using the paginator method of operation (using the paginator helper)
* being able iterate over empty descriptions, by handling the Description issue correctly (and defaulting to empty string)

- [X] By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: src/lambda_codebase/moved_to_root.py [hunk 1]
```diff
@@ -28,9 +29,10 @@ def worker_thread(sts, region, account_id, role, event):
         'remove_base')
 
     parameter_store = ParameterStore(region, role)
-    parameters = [param['Name'] for param in parameter_store.client.describe_parameters()['Parameters'] if 'Used by The AWS Deployment Framework' in param['Description']]
-    for parameter in parameters:
-        parameter_store.delete_parameter(parameter)
+    for page in paginator(parameter_store.client.describe_parameters):
```
Comment (sbkok) [inline src/lambda_codebase/moved_to_root.py:32]: Could you please change this to use the Boto3 paginator instead?
See the docs in boto3 here: https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html

**Review by sbkok — CHANGES_REQUESTED** [review]

File: src/lambda_codebase/moved_to_root.py (hunk 1 above)
Comment (rickardl) [reply src/lambda_codebase/moved_to_root.py:32]: Absolutely, I'll update the PR next week!

**Review by sbkok — APPROVED** [review]
Thanks for updating the pull request and contributing this fix.
Looks good to me.

**Review by deltagarrett — APPROVED** [review]
LGTM

**Pull request merged**